[TikTok] ERROR: Postprocessing: Conversion failed! when embedding thumbnail

### DO NOT REMOVE OR SKIP THE ISSUE TEMPLATE

- [X] I understand that I will be **blocked** if I *intentionally* remove or skip any mandatory\* field

### Checklist

- [X] I'm reporting that yt-dlp is broken on a **supported** site
- [X] I've verified that I have **updated yt-dlp to nightly or master** ([update instructions](https://github.com/yt-dlp/yt-dlp#update-channels))
- [X] I've checked that all provided URLs are playable in a browser with the same IP and same login details
- [X] I've checked that all URLs and arguments with special characters are [properly quoted or escaped](https://github.com/yt-dlp/yt-dlp/wiki/FAQ#video-url-contains-an-ampersand--and-im-getting-some-strange-output-1-2839-or-v-is-not-recognized-as-an-internal-or-external-command)
- [X] I've searched [known issues](https://github.com/yt-dlp/yt-dlp/issues/3766) and the [bugtracker](https://github.com/yt-dlp/yt-dlp/issues?q=) for similar issues **including closed ones**. DO NOT post duplicates
- [X] I've read the [guidelines for opening an issue](https://github.com/yt-dlp/yt-dlp/blob/master/CONTRIBUTING.md#opening-an-issue)
- [ ] I've read about [sharing account credentials](https://github.com/yt-dlp/yt-dlp/blob/master/CONTRIBUTING.md#are-you-willing-to-share-account-details-if-needed) and I'm willing to share it if required

### Region

_No response_

### Provide a description that is worded well enough to be understood

Mainly the error "conversion failed", post processor errors. Lots of videos don't download. Also errors that have to do with "skipping unsupported chunk: ANMF" and "Nothing was written into output file, because at least one of its streams received no packets. Conversion failed!"

### Provide verbose output that clearly demonstrates the problem

- [X] Run **your** yt-dlp command with **-vU** flag added (`yt-dlp -vU <your command line>`)
- [ ] If using API, add `'verbose': True` to `YoutubeDL` params instead
- [X] Copy the WHOLE output (starting with `[debug] Command-line config`) and insert it below

### Complete Verbose Output

```shell
[debug] Command-line config: ['https://www.tiktok.com/@cooperspamsasf/video/7432045283686632710', '--download-archive', 'F:\\yt-dlp tiktok likes\\archive.txt', '--write-thumbnail', '--embed-thumbnail', '--verbose', '--cookies-from-browser', 'firefox']
[debug] Encodings: locale cp1252, fs utf-8, pref cp1252, out utf-8, error cp1252 (No VT), screen utf-8
[debug] yt-dlp version stable@2024.11.18 from yt-dlp/yt-dlp [7ea278792] (pip)
[debug] Python 3.11.9 (CPython AMD64 64bit) - Windows-10-10.0.22631-SP0 (OpenSSL 3.3.2 3 Sep 2024)
[debug] exe versions: ffmpeg 7.0.2-full_build-www.gyan.dev (setts), ffprobe 7.0.2-full_build-www.gyan.dev
[debug] Optional libraries: Cryptodome-3.20.0, brotli-1.0.9, certifi-2024.08.30, curl_cffi-0.5.10, mutagen-1.47.0, requests-2.32.3, sqlite3-3.45.3, urllib3-2.2.3, websockets-13.1
[debug] Proxy map: {}
[debug] Extracting cookies from: "C:\Users\J\AppData\Roaming\Mozilla\Firefox\Profiles\c2ty66d6.default-release\cookies.sqlite"
[debug] Request Handlers: urllib, requests, websockets, curl_cffi
[debug] Loaded 1837 extractors
[debug] Loading archive file 'F:\\yt-dlp tiktok likes\\archive.txt'
[debug] [TikTok] Found universal data for rehydration
[debug] Formats sorted by: hasvid, ie_pref, lang, quality, res, fps, hdr:12(7), vcodec, channels, acodec, size, br, asr, proto, vext, aext, hasaud, source, id
[debug] Default format spec: bestvideo*+bestaudio/best
[debug] Invoking http downloader on "https://v19-webapp-prime.tiktok.com/video/tos/useast2a/tos-useast2a-pve-0068/oAXOvcjeEAZzgjgfgQLKR5SGzeNrxA9ICICxHI/?a=1988&bti=ODszNWYuMDE6&ch=0&cr=3&dr=0&lr=all&cd=0%7C0%7C0%7C&cv=1&br=2404&bt=1202&cs=2&ds=4&ft=4fUEKMk88Zmo0WRLZb4jVaThrpWrKsd.&mime_type=video_mp4&qs=15&rc=NzNpZWU8OzRmNzs0Nzk1aUBpam93dnY5cnh4djMzNzczM0AtMS0uNS41NTIxMTBhXzEyYSNmZW9uMmRjbGVgLS1kMTZzcw%3D%3D&btag=e00088000&expire=1732609535&l=2024112602251903ACD4E62348E641B01E&ply_type=2&policy=2&signature=1e746658933c8ee3a81756c4afee15d3&tk=tt_chain_token"
[debug] ffmpeg command line: ffmpeg -y -loglevel repeat+info -f image2 -pattern_type none -i "file:HALLOWEEN TYPEE  #halloweencostume #swat #duo #blonde #brunette #thatassperfectbaby #soccergirls  [7432045283686632710].webp" -update 1 -movflags +faststart "file:HALLOWEEN TYPEE  #halloweencostume #swat #duo #blonde #brunette #thatassperfectbaby #soccergirls  [7432045283686632710].png"
[debug] ffmpeg version 7.0.2-full_build-www.gyan.dev Copyright (c) 2000-2024 the FFmpeg developers
  built with gcc 13.2.0 (Rev5, Built by MSYS2 project)
  configuration: --enable-gpl --enable-version3 --enable-static --disable-w32threads --disable-autodetect --enable-fontconfig --enable-iconv --enable-gnutls --enable-libxml2 --enable-gmp --enable-bzlib --enable-lzma --enable-libsnappy --enable-zlib --enable-librist --enable-libsrt --enable-libssh --enable-libzmq --enable-avisynth --enable-libbluray --enable-libcaca --enable-sdl2 --enable-libaribb24 --enable-libaribcaption --enable-libdav1d --enable-libdavs2 --enable-libuavs3d --enable-libxevd --enable-libzvbi --enable-librav1e --enable-libsvtav1 --enable-libwebp --enable-libx264 --enable-libx265 --enable-libxavs2 --enable-libxeve --enable-libxvid --enable-libaom --enable-libjxl --enable-libopenjpeg --enable-libvpx --enable-mediafoundation --enable-libass --enable-frei0r --enable-libfreetype --enable-libfribidi --enable-libharfbuzz --enable-liblensfun --enable-libvidstab --enable-libvmaf --enable-libzimg --enable-amf --enable-cuda-llvm --enable-cuvid --enable-dxva2 --enable-d3d11va --enable-d3d12va --enable-ffnvcodec --enable-libvpl --enable-nvdec --enable-nvenc --enable-vaapi --enable-libshaderc --enable-vulkan --enable-libplacebo --enable-opencl --enable-libcdio --enable-libgme --enable-libmodplug --enable-libopenmpt --enable-libopencore-amrwb --enable-libmp3lame --enable-libshine --enable-libtheora --enable-libtwolame --enable-libvo-amrwbenc --enable-libcodec2 --enable-libilbc --enable-libgsm --enable-libopencore-amrnb --enable-libopus --enable-libspeex --enable-libvorbis --enable-ladspa --enable-libbs2b --enable-libflite --enable-libmysofa --enable-librubberband --enable-libsoxr --enable-chromaprint
  libavutil      59.  8.100 / 59.  8.100
  libavcodec     61.  3.100 / 61.  3.100
  libavformat    61.  1.100 / 61.  1.100
  libavdevice    61.  1.100 / 61.  1.100
  libavfilter    10.  1.100 / 10.  1.100
  libswscale      8.  1.100 /  8.  1.100
  libswresample   5.  1.100 /  5.  1.100
  libpostproc    58.  1.100 / 58.  1.100
[webp @ 000001545c4432c0] skipping unsupported chunk: ANIM
[webp @ 000001545c4432c0] skipping unsupported chunk: ANMF
[webp @ 000001545c4432c0] skipping unsupported chunk: ANMF
[webp @ 000001545c4432c0] skipping unsupported chunk: ANMF
[webp @ 000001545c4432c0] skipping unsupported chunk: ANMF
[webp @ 000001545c4432c0] skipping unsupported chunk: ANMF
[webp @ 000001545c4432c0] skipping unsupported chunk: ANMF
[webp @ 000001545c4432c0] skipping unsupported chunk: ANMF
[webp @ 000001545c4432c0] skipping unsupported chunk: ANMF
[webp @ 000001545c4432c0] skipping unsupported chunk: ANMF
[webp @ 000001545c4432c0] image data not found
[image2 @ 000001545c441940] Could not find codec parameters for stream 0 (Video: webp, none): unspecified size
Consider increasing the value for the 'analyzeduration' (0) and 'probesize' (5000000) options
Input #0, image2, from 'file:HALLOWEEN TYPEE  #halloweencostume #swat #duo #blonde #brunette #thatassperfectbaby #soccergirls  [7432045283686632710].webp':
  Duration: 00:00:00.04, start: 0.000000, bitrate: N/A
  Stream #0:0: Video: webp, none, 25 fps, 25 tbr, 25 tbn
Stream mapping:
  Stream #0:0 -> #0:0 (webp (native) -> png (native))
Press [q] to stop, [?] for help
[webp @ 000001545c469fc0] skipping unsupported chunk: ANIM
[webp @ 000001545c469fc0] skipping unsupported chunk: ANMF
[webp @ 000001545c469fc0] skipping unsupported chunk: ANMF
[webp @ 000001545c469fc0] skipping unsupported chunk: ANMF
[webp @ 000001545c469fc0] skipping unsupported chunk: ANMF
[webp @ 000001545c469fc0] skipping unsupported chunk: ANMF
[webp @ 000001545c469fc0] skipping unsupported chunk: ANMF
[webp @ 000001545c469fc0] skipping unsupported chunk: ANMF
[webp @ 000001545c469fc0] skipping unsupported chunk: ANMF
[webp @ 000001545c469fc0] skipping unsupported chunk: ANMF
[webp @ 000001545c469fc0] image data not found
[vist#0:0/webp @ 000001545c4432c0] [dec:webp @ 000001545c44c440] Decoding error: Invalid data found when processing input
[vist#0:0/webp @ 000001545c4432c0] [dec:webp @ 000001545c44c440] Decode error rate 1 exceeds maximum 0.666667
[vist#0:0/webp @ 000001545c4432c0] [dec:webp @ 000001545c44c440] Task finished with error code: -1145393733 (Error number -1145393733 occurred)
[vist#0:0/webp @ 000001545c4432c0] [dec:webp @ 000001545c44c440] Terminating thread with return code -1145393733 (Error number -1145393733 occurred)
Cannot determine format of input 0:0 after EOF
[vf#0:0 @ 000001545c44ac80] Task finished with error code: -1094995529 (Invalid data found when processing input)
[vf#0:0 @ 000001545c44ac80] Terminating thread with return code -1094995529 (Invalid data found when processing input)
[vost#0:0/png @ 000001545c448c00] Could not open encoder before EOF
[vost#0:0/png @ 000001545c448c00] Task finished with error code: -22 (Invalid argument)
[vost#0:0/png @ 000001545c448c00] Terminating thread with return code -22 (Invalid argument)
[out#0/image2 @ 000001545c467e40] Nothing was written into output file, because at least one of its streams received no packets.
frame=    0 fps=0.0 q=0.0 Lsize=       0KiB time=N/A bitrate=N/A speed=N/A    
Conversion failed!

ERROR: Postprocessing: Conversion failed!
Traceback (most recent call last):
  File "C:\Users\J\miniconda3\Lib\site-packages\yt_dlp\YoutubeDL.py", line 3556, in process_info
    replace_info_dict(self.post_process(dl_filename, info_dict, files_to_move))
                      ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\J\miniconda3\Lib\site-packages\yt_dlp\YoutubeDL.py", line 3740, in post_process
    info = self.run_all_pps('post_process', info, additional_pps=info.get('__postprocessors'))
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\J\miniconda3\Lib\site-packages\yt_dlp\YoutubeDL.py", line 3722, in run_all_pps
    info = self.run_pp(pp, info)
           ^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\J\miniconda3\Lib\site-packages\yt_dlp\YoutubeDL.py", line 3700, in run_pp
    files_to_delete, infodict = pp.run(infodict)
                                ^^^^^^^^^^^^^^^^
  File "C:\Users\J\miniconda3\Lib\site-packages\yt_dlp\postprocessor\common.py", line 22, in run
    ret = func(self, info, *args, **kwargs)
          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\J\miniconda3\Lib\site-packages\yt_dlp\postprocessor\common.py", line 127, in wrapper
    return func(self, info)
           ^^^^^^^^^^^^^^^^
  File "C:\Users\J\miniconda3\Lib\site-packages\yt_dlp\postprocessor\embedthumbnail.py", line 84, in run
    thumbnail_filename = convertor.convert_thumbnail(thumbnail_filename, 'png')
                         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\J\miniconda3\Lib\site-packages\yt_dlp\postprocessor\ffmpeg.py", line 1107, in convert_thumbnail
    self.real_run_ffmpeg(
  File "C:\Users\J\miniconda3\Lib\site-packages\yt_dlp\postprocessor\ffmpeg.py", line 367, in real_run_ffmpeg
    raise FFmpegPostProcessorError(stderr.strip().splitlines()[-1])
yt_dlp.postprocessor.ffmpeg.FFmpegPostProcessorError: Conversion failed!

[ERROR] Failed to process URL: https://www.tiktok.com/@cooperspamsasf/video/7432045283686632710 
[debug] Command-line config: ['https://www.tiktok.com/@bris.main/video/7439516415444536606', '--download-archive', 'F:\\yt-dlp tiktok likes\\archive.txt', '--write-thumbnail', '--embed-thumbnail', '--verbose', '--cookies-from-browser', 'firefox']
[debug] Encodings: locale cp1252, fs utf-8, pref cp1252, out utf-8, error cp1252 (No VT), screen utf-8
[debug] yt-dlp version stable@2024.11.18 from yt-dlp/yt-dlp [7ea278792] (pip)
[debug] Python 3.11.9 (CPython AMD64 64bit) - Windows-10-10.0.22631-SP0 (OpenSSL 3.3.2 3 Sep 2024)
[debug] exe versions: ffmpeg 7.0.2-full_build-www.gyan.dev (setts), ffprobe 7.0.2-full_build-www.gyan.dev
[debug] Optional libraries: Cryptodome-3.20.0, brotli-1.0.9, certifi-2024.08.30, curl_cffi-0.5.10, mutagen-1.47.0, requests-2.32.3, sqlite3-3.45.3, urllib3-2.2.3, websockets-13.1
[debug] Proxy map: {}
[debug] Extracting cookies from: "C:\Users\J\AppData\Roaming\Mozilla\Firefox\Profiles\c2ty66d6.default-release\cookies.sqlite"
[debug] Request Handlers: urllib, requests, websockets, curl_cffi
[debug] Loaded 1837 extractors
[debug] Loading archive file 'F:\\yt-dlp tiktok likes\\archive.txt'
[debug] Command-line config: ['https://www.tiktok.com/@leilaaaaaaaaa34/video/7430073853495299350', '--download-archive', 'F:\\yt-dlp tiktok likes\\archive.txt', '--write-thumbnail', '--embed-thumbnail', '--verbose', '--cookies-from-browser', 'firefox']
[debug] Encodings: locale cp1252, fs utf-8, pref cp1252, out utf-8, error cp1252 (No VT), screen utf-8
[debug] yt-dlp version stable@2024.11.18 from yt-dlp/yt-dlp [7ea278792] (pip)
[debug] Python 3.11.9 (CPython AMD64 64bit) - Windows-10-10.0.22631-SP0 (OpenSSL 3.3.2 3 Sep 2024)
[debug] exe versions: ffmpeg 7.0.2-full_build-www.gyan.dev (setts), ffprobe 7.0.2-full_build-www.gyan.dev
[debug] Optional libraries: Cryptodome-3.20.0, brotli-1.0.9, certifi-2024.08.30, curl_cffi-0.5.10, mutagen-1.47.0, requests-2.32.3, sqlite3-3.45.3, urllib3-2.2.3, websockets-13.1
[debug] Proxy map: {}
[debug] Extracting cookies from: "C:\Users\J\AppData\Roaming\Mozilla\Firefox\Profiles\c2ty66d6.default-release\cookies.sqlite"
[debug] Request Handlers: urllib, requests, websockets, curl_cffi
[debug] Loaded 1837 extractors
[debug] Loading archive file 'F:\\yt-dlp tiktok likes\\archive.txt'
[debug] Command-line config: ['https://www.tiktok.com/@erindottie/video/7428505324375559457', '--download-archive', 'F:\\yt-dlp tiktok likes\\archive.txt', '--write-thumbnail', '--embed-thumbnail', '--verbose', '--cookies-from-browser', 'firefox']
[debug] Encodings: locale cp1252, fs utf-8, pref cp1252, out utf-8, error cp1252 (No VT), screen utf-8
[debug] yt-dlp version stable@2024.11.18 from yt-dlp/yt-dlp [7ea278792] (pip)
[debug] Python 3.11.9 (CPython AMD64 64bit) - Windows-10-10.0.22631-SP0 (OpenSSL 3.3.2 3 Sep 2024)
[debug] exe versions: ffmpeg 7.0.2-full_build-www.gyan.dev (setts), ffprobe 7.0.2-full_build-www.gyan.dev
[debug] Optional libraries: Cryptodome-3.20.0, brotli-1.0.9, certifi-2024.08.30, curl_cffi-0.5.10, mutagen-1.47.0, requests-2.32.3, sqlite3-3.45.3, urllib3-2.2.3, websockets-13.1
[debug] Proxy map: {}
[debug] Extracting cookies from: "C:\Users\J\AppData\Roaming\Mozilla\Firefox\Profiles\c2ty66d6.default-release\cookies.sqlite"
[debug] Request Handlers: urllib, requests, websockets, curl_cffi
[debug] Loaded 1837 extractors
[debug] Loading archive file 'F:\\yt-dlp tiktok likes\\archive.txt'
[debug] Command-line config: ['https://www.tiktok.com/@user415387491623/video/7434688554627910968', '--download-archive', 'F:\\yt-dlp tiktok likes\\archive.txt', '--write-thumbnail', '--embed-thumbnail', '--verbose', '--cookies-from-browser', 'firefox']
[debug] Encodings: locale cp1252, fs utf-8, pref cp1252, out utf-8, error cp1252 (No VT), screen utf-8
[debug] yt-dlp version stable@2024.11.18 from yt-dlp/yt-dlp [7ea278792] (pip)
[debug] Python 3.11.9 (CPython AMD64 64bit) - Windows-10-10.0.22631-SP0 (OpenSSL 3.3.2 3 Sep 2024)
[debug] exe versions: ffmpeg 7.0.2-full_build-www.gyan.dev (setts), ffprobe 7.0.2-full_build-www.gyan.dev
[debug] Optional libraries: Cryptodome-3.20.0, brotli-1.0.9, certifi-2024.08.30, curl_cffi-0.5.10, mutagen-1.47.0, requests-2.32.3, sqlite3-3.45.3, urllib3-2.2.3, websockets-13.1
[debug] Proxy map: {}
[debug] Extracting cookies from: "C:\Users\J\AppData\Roaming\Mozilla\Firefox\Profiles\c2ty66d6.default-release\cookies.sqlite"
[debug] Request Handlers: urllib, requests, websockets, curl_cffi
[debug] Loaded 1837 extractors
[debug] Loading archive file 'F:\\yt-dlp tiktok likes\\archive.txt'
[debug] Command-line config: ['https://www.tiktok.com/@elsa.vikstrom/video/7431528033044942102', '--download-archive', 'F:\\yt-dlp tiktok likes\\archive.txt', '--write-thumbnail', '--embed-thumbnail', '--verbose', '--cookies-from-browser', 'firefox']
[debug] Encodings: locale cp1252, fs utf-8, pref cp1252, out utf-8, error cp1252 (No VT), screen utf-8
[debug] yt-dlp version stable@2024.11.18 from yt-dlp/yt-dlp [7ea278792] (pip)
[debug] Python 3.11.9 (CPython AMD64 64bit) - Windows-10-10.0.22631-SP0 (OpenSSL 3.3.2 3 Sep 2024)
[debug] exe versions: ffmpeg 7.0.2-full_build-www.gyan.dev (setts), ffprobe 7.0.2-full_build-www.gyan.dev
[debug] Optional libraries: Cryptodome-3.20.0, brotli-1.0.9, certifi-2024.08.30, curl_cffi-0.5.10, mutagen-1.47.0, requests-2.32.3, sqlite3-3.45.3, urllib3-2.2.3, websockets-13.1
[debug] Proxy map: {}
[debug] Extracting cookies from: "C:\Users\J\AppData\Roaming\Mozilla\Firefox\Profiles\c2ty66d6.default-release\cookies.sqlite"
[debug] Request Handlers: urllib, requests, websockets, curl_cffi
[debug] Loaded 1837 extractors
[debug] Loading archive file 'F:\\yt-dlp tiktok likes\\archive.txt'
[debug] Command-line config: ['https://www.tiktok.com/@ellatomine2/video/7440197178603228449', '--download-archive', 'F:\\yt-dlp tiktok likes\\archive.txt', '--write-thumbnail', '--embed-thumbnail', '--verbose', '--cookies-from-browser', 'firefox']
[debug] Encodings: locale cp1252, fs utf-8, pref cp1252, out utf-8, error cp1252 (No VT), screen utf-8
[debug] yt-dlp version stable@2024.11.18 from yt-dlp/yt-dlp [7ea278792] (pip)
[debug] Python 3.11.9 (CPython AMD64 64bit) - Windows-10-10.0.22631-SP0 (OpenSSL 3.3.2 3 Sep 2024)
[debug] exe versions: ffmpeg 7.0.2-full_build-www.gyan.dev (setts), ffprobe 7.0.2-full_build-www.gyan.dev
[debug] Optional libraries: Cryptodome-3.20.0, brotli-1.0.9, certifi-2024.08.30, curl_cffi-0.5.10, mutagen-1.47.0, requests-2.32.3, sqlite3-3.45.3, urllib3-2.2.3, websockets-13.1
[debug] Proxy map: {}
[debug] Extracting cookies from: "C:\Users\J\AppData\Roaming\Mozilla\Firefox\Profiles\c2ty66d6.default-release\cookies.sqlite"
[debug] Request Handlers: urllib, requests, websockets, curl_cffi
[debug] Loaded 1837 extractors
[debug] Loading archive file 'F:\\yt-dlp tiktok likes\\archive.txt'
[debug] Command-line config: ['https://www.tiktok.com/@elena__blondie/video/7440396119076506912', '--download-archive', 'F:\\yt-dlp tiktok likes\\archive.txt', '--write-thumbnail', '--embed-thumbnail', '--verbose', '--cookies-from-browser', 'firefox']
[debug] Encodings: locale cp1252, fs utf-8, pref cp1252, out utf-8, error cp1252 (No VT), screen utf-8
[debug] yt-dlp version stable@2024.11.18 from yt-dlp/yt-dlp [7ea278792] (pip)
[debug] Python 3.11.9 (CPython AMD64 64bit) - Windows-10-10.0.22631-SP0 (OpenSSL 3.3.2 3 Sep 2024)
[debug] exe versions: ffmpeg 7.0.2-full_build-www.gyan.dev (setts), ffprobe 7.0.2-full_build-www.gyan.dev
[debug] Optional libraries: Cryptodome-3.20.0, brotli-1.0.9, certifi-2024.08.30, curl_cffi-0.5.10, mutagen-1.47.0, requests-2.32.3, sqlite3-3.45.3, urllib3-2.2.3, websockets-13.1
[debug] Proxy map: {}
[debug] Extracting cookies from: "C:\Users\J\AppData\Roaming\Mozilla\Firefox\Profiles\c2ty66d6.default-release\cookies.sqlite"
[debug] Request Handlers: urllib, requests, websockets, curl_cffi
[debug] Loaded 1837 extractors
[debug] Loading archive file 'F:\\yt-dlp tiktok likes\\archive.txt'
[debug] Command-line config: ['https://www.tiktok.com/@johaanssson/video/7440864222747086112', '--download-archive', 'F:\\yt-dlp tiktok likes\\archive.txt', '--write-thumbnail', '--embed-thumbnail', '--verbose', '--cookies-from-browser', 'firefox']
[debug] Encodings: locale cp1252, fs utf-8, pref cp1252, out utf-8, error cp1252 (No VT), screen utf-8
[debug] yt-dlp version stable@2024.11.18 from yt-dlp/yt-dlp [7ea278792] (pip)
[debug] Python 3.11.9 (CPython AMD64 64bit) - Windows-10-10.0.22631-SP0 (OpenSSL 3.3.2 3 Sep 2024)
[debug] exe versions: ffmpeg 7.0.2-full_build-www.gyan.dev (setts), ffprobe 7.0.2-full_build-www.gyan.dev
[debug] Optional libraries: Cryptodome-3.20.0, brotli-1.0.9, certifi-2024.08.30, curl_cffi-0.5.10, mutagen-1.47.0, requests-2.32.3, sqlite3-3.45.3, urllib3-2.2.3, websockets-13.1
[debug] Proxy map: {}
[debug] Extracting cookies from: "C:\Users\J\AppData\Roaming\Mozilla\Firefox\Profiles\c2ty66d6.default-release\cookies.sqlite"
[debug] Request Handlers: urllib, requests, websockets, curl_cffi
[debug] Loaded 1837 extractors
[debug] Loading archive file 'F:\\yt-dlp tiktok likes\\archive.txt'
[debug] [TikTok] Found universal data for rehydration
[debug] Formats sorted by: hasvid, ie_pref, lang, quality, res, fps, hdr:12(7), vcodec, channels, acodec, size, br, asr, proto, vext, aext, hasaud, source, id
[debug] Default format spec: bestvideo*+bestaudio/best
[debug] Invoking http downloader on "https://v19-webapp-prime.tiktok.com/video/tos/useast2a/tos-useast2a-ve-0068-euttp/ok6GJnAQE2q0AFfyAaPQIQDhK0KQBwD1EIcfR4/?a=1988&bti=ODszNWYuMDE6&ch=0&cr=3&dr=0&lr=all&cd=0%7C0%7C0%7C&cv=1&br=1346&bt=673&cs=2&ds=4&eid=256&ft=4fUEKMk88Zmo0bRLZb4jVHCurpWrKsd.&mime_type=video_mp4&qs=15&rc=ZDVnOzplaWlpZzdmNmdpOUBpM3ZuM3Q5cndudzMzZjczM0AxY2A0LzZjNTMxLTAwY2JfYSNgLTBoMmQ0MS5gLS1kMWNzcw%3D%3D&btag=e00088000&expire=1732609546&l=202411260225363935CAF2808D524710A5&ply_type=2&policy=2&signature=c15a759aebb22c7a55843e0c19030be4&tk=tt_chain_token"
[debug] ffmpeg command line: ffmpeg -y -loglevel repeat+info -f image2 -pattern_type none -i "file:Ghettooo #fyp #viral #trend  [7440864222747086112].webp" -update 1 -movflags +faststart "file:Ghettooo #fyp #viral #trend  [7440864222747086112].png"
[debug] ffmpeg version 7.0.2-full_build-www.gyan.dev Copyright (c) 2000-2024 the FFmpeg developers
  built with gcc 13.2.0 (Rev5, Built by MSYS2 project)
  configuration: --enable-gpl --enable-version3 --enable-static --disable-w32threads --disable-autodetect --enable-fontconfig --enable-iconv --enable-gnutls --enable-libxml2 --enable-gmp --enable-bzlib --enable-lzma --enable-libsnappy --enable-zlib --enable-librist --enable-libsrt --enable-libssh --enable-libzmq --enable-avisynth --enable-libbluray --enable-libcaca --enable-sdl2 --enable-libaribb24 --enable-libaribcaption --enable-libdav1d --enable-libdavs2 --enable-libuavs3d --enable-libxevd --enable-libzvbi --enable-librav1e --enable-libsvtav1 --enable-libwebp --enable-libx264 --enable-libx265 --enable-libxavs2 --enable-libxeve --enable-libxvid --enable-libaom --enable-libjxl --enable-libopenjpeg --enable-libvpx --enable-mediafoundation --enable-libass --enable-frei0r --enable-libfreetype --enable-libfribidi --enable-libharfbuzz --enable-liblensfun --enable-libvidstab --enable-libvmaf --enable-libzimg --enable-amf --enable-cuda-llvm --enable-cuvid --enable-dxva2 --enable-d3d11va --enable-d3d12va --enable-ffnvcodec --enable-libvpl --enable-nvdec --enable-nvenc --enable-vaapi --enable-libshaderc --enable-vulkan --enable-libplacebo --enable-opencl --enable-libcdio --enable-libgme --enable-libmodplug --enable-libopenmpt --enable-libopencore-amrwb --enable-libmp3lame --enable-libshine --enable-libtheora --enable-libtwolame --enable-libvo-amrwbenc --enable-libcodec2 --enable-libilbc --enable-libgsm --enable-libopencore-amrnb --enable-libopus --enable-libspeex --enable-libvorbis --enable-ladspa --enable-libbs2b --enable-libflite --enable-libmysofa --enable-librubberband --enable-libsoxr --enable-chromaprint
  libavutil      59.  8.100 / 59.  8.100
  libavcodec     61.  3.100 / 61.  3.100
  libavformat    61.  1.100 / 61.  1.100
  libavdevice    61.  1.100 / 61.  1.100
  libavfilter    10.  1.100 / 10.  1.100
  libswscale      8.  1.100 /  8.  1.100
  libswresample   5.  1.100 /  5.  1.100
  libpostproc    58.  1.100 / 58.  1.100
[webp @ 000001c6cbb512c0] skipping unsupported chunk: ANIM
[webp @ 000001c6cbb512c0] skipping unsupported chunk: ANMF
[webp @ 000001c6cbb512c0] skipping unsupported chunk: ANMF
[webp @ 000001c6cbb512c0] skipping unsupported chunk: ANMF
[webp @ 000001c6cbb512c0] skipping unsupported chunk: ANMF
[webp @ 000001c6cbb512c0] skipping unsupported chunk: ANMF
[webp @ 000001c6cbb512c0] skipping unsupported chunk: ANMF
[webp @ 000001c6cbb512c0] skipping unsupported chunk: ANMF
[webp @ 000001c6cbb512c0] skipping unsupported chunk: ANMF
[webp @ 000001c6cbb512c0] skipping unsupported chunk: ANMF
[webp @ 000001c6cbb512c0] image data not found
[image2 @ 000001c6cbb569c0] Could not find codec parameters for stream 0 (Video: webp, none): unspecified size
Consider increasing the value for the 'analyzeduration' (0) and 'probesize' (5000000) options
Input #0, image2, from 'file:Ghettooo #fyp #viral #trend  [7440864222747086112].webp':
  Duration: 00:00:00.04, start: 0.000000, bitrate: N/A
  Stream #0:0: Video: webp, none, 25 fps, 25 tbr, 25 tbn
Stream mapping:
  Stream #0:0 -> #0:0 (webp (native) -> png (native))
Press [q] to stop, [?] for help
[webp @ 000001c6cbb61cc0] skipping unsupported chunk: ANIM
[webp @ 000001c6cbb61cc0] skipping unsupported chunk: ANMF
[webp @ 000001c6cbb61cc0] skipping unsupported chunk: ANMF
[webp @ 000001c6cbb61cc0] skipping unsupported chunk: ANMF
[webp @ 000001c6cbb61cc0] skipping unsupported chunk: ANMF
[webp @ 000001c6cbb61cc0] skipping unsupported chunk: ANMF
[webp @ 000001c6cbb61cc0] skipping unsupported chunk: ANMF
[webp @ 000001c6cbb61cc0] skipping unsupported chunk: ANMF
[webp @ 000001c6cbb61cc0] skipping unsupported chunk: ANMF
[webp @ 000001c6cbb61cc0] skipping unsupported chunk: ANMF
[webp @ 000001c6cbb61cc0] image data not found
[vist#0:0/webp @ 000001c6cbb512c0] [dec:webp @ 000001c6cbb59e00] Decoding error: Invalid data found when processing input
[vist#0:0/webp @ 000001c6cbb512c0] [dec:webp @ 000001c6cbb59e00] Decode error rate 1 exceeds maximum 0.666667
[vist#0:0/webp @ 000001c6cbb512c0] [dec:webp @ 000001c6cbb59e00] Task finished with error code: -1145393733 (Error number -1145393733 occurred)
[vist#0:0/webp @ 000001c6cbb512c0] [dec:webp @ 000001c6cbb59e00] Terminating thread with return code -1145393733 (Error number -1145393733 occurred)
Cannot determine format of input 0:0 after EOF
[vf#0:0 @ 000001c6cbb53140] Task finished with error code: -1094995529 (Invalid data found when processing input)
[vf#0:0 @ 000001c6cbb53140] Terminating thread with return code -1094995529 (Invalid data found when processing input)
[vost#0:0/png @ 000001c6cbb6f7c0] Could not open encoder before EOF
[vost#0:0/png @ 000001c6cbb6f7c0] Task finished with error code: -22 (Invalid argument)
[vost#0:0/png @ 000001c6cbb6f7c0] Terminating thread with return code -22 (Invalid argument)
[out#0/image2 @ 000001c6cbb6ef40] Nothing was written into output file, because at least one of its streams received no packets.
frame=    0 fps=0.0 q=0.0 Lsize=       0KiB time=N/A bitrate=N/A speed=N/A    
Conversion failed!

ERROR: Postprocessing: Conversion failed!
Traceback (most recent call last):
  File "C:\Users\J\miniconda3\Lib\site-packages\yt_dlp\YoutubeDL.py", line 3556, in process_info
    replace_info_dict(self.post_process(dl_filename, info_dict, files_to_move))
                      ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\J\miniconda3\Lib\site-packages\yt_dlp\YoutubeDL.py", line 3740, in post_process
    info = self.run_all_pps('post_process', info, additional_pps=info.get('__postprocessors'))
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\J\miniconda3\Lib\site-packages\yt_dlp\YoutubeDL.py", line 3722, in run_all_pps
    info = self.run_pp(pp, info)
           ^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\J\miniconda3\Lib\site-packages\yt_dlp\YoutubeDL.py", line 3700, in run_pp
    files_to_delete, infodict = pp.run(infodict)
                                ^^^^^^^^^^^^^^^^
  File "C:\Users\J\miniconda3\Lib\site-packages\yt_dlp\postprocessor\common.py", line 22, in run
    ret = func(self, info, *args, **kwargs)
          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\J\miniconda3\Lib\site-packages\yt_dlp\postprocessor\common.py", line 127, in wrapper
    return func(self, info)
           ^^^^^^^^^^^^^^^^
  File "C:\Users\J\miniconda3\Lib\site-packages\yt_dlp\postprocessor\embedthumbnail.py", line 84, in run
    thumbnail_filename = convertor.convert_thumbnail(thumbnail_filename, 'png')
                         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\J\miniconda3\Lib\site-packages\yt_dlp\postprocessor\ffmpeg.py", line 1107, in convert_thumbnail
    self.real_run_ffmpeg(
  File "C:\Users\J\miniconda3\Lib\site-packages\yt_dlp\postprocessor\ffmpeg.py", line 367, in real_run_ffmpeg
    raise FFmpegPostProcessorError(stderr.strip().splitlines()[-1])
yt_dlp.postprocessor.ffmpeg.FFmpegPostProcessorError: Conversion failed!

[ERROR] Failed to process URL: https://www.tiktok.com/@johaanssson/video/7440864222747086112 
[debug] Command-line config: ['https://www.tiktok.com/@filippasekesan0/video/7440543183844560150', '--download-archive', 'F:\\yt-dlp tiktok likes\\archive.txt', '--write-thumbnail', '--embed-thumbnail', '--verbose', '--cookies-from-browser', 'firefox']
[debug] Encodings: locale cp1252, fs utf-8, pref cp1252, out utf-8, error cp1252 (No VT), screen utf-8
[debug] yt-dlp version stable@2024.11.18 from yt-dlp/yt-dlp [7ea278792] (pip)
[debug] Python 3.11.9 (CPython AMD64 64bit) - Windows-10-10.0.22631-SP0 (OpenSSL 3.3.2 3 Sep 2024)
[debug] exe versions: ffmpeg 7.0.2-full_build-www.gyan.dev (setts), ffprobe 7.0.2-full_build-www.gyan.dev
[debug] Optional libraries: Cryptodome-3.20.0, brotli-1.0.9, certifi-2024.08.30, curl_cffi-0.5.10, mutagen-1.47.0, requests-2.32.3, sqlite3-3.45.3, urllib3-2.2.3, websockets-13.1
[debug] Proxy map: {}
[debug] Extracting cookies from: "C:\Users\J\AppData\Roaming\Mozilla\Firefox\Profiles\c2ty66d6.default-release\cookies.sqlite"
[debug] Request Handlers: urllib, requests, websockets, curl_cffi
[debug] Loaded 1837 extractors
[debug] Loading archive file 'F:\\yt-dlp tiktok likes\\archive.txt'
[debug] Command-line config: ['https://www.tiktok.com/@elana.maguire15/video/7439872632234708257', '--download-archive', 'F:\\yt-dlp tiktok likes\\archive.txt', '--write-thumbnail', '--embed-thumbnail', '--verbose', '--cookies-from-browser', 'firefox']
[debug] Encodings: locale cp1252, fs utf-8, pref cp1252, out utf-8, error cp1252 (No VT), screen utf-8
[debug] yt-dlp version stable@2024.11.18 from yt-dlp/yt-dlp [7ea278792] (pip)
[debug] Python 3.11.9 (CPython AMD64 64bit) - Windows-10-10.0.22631-SP0 (OpenSSL 3.3.2 3 Sep 2024)
[debug] exe versions: ffmpeg 7.0.2-full_build-www.gyan.dev (setts), ffprobe 7.0.2-full_build-www.gyan.dev
[debug] Optional libraries: Cryptodome-3.20.0, brotli-1.0.9, certifi-2024.08.30, curl_cffi-0.5.10, mutagen-1.47.0, requests-2.32.3, sqlite3-3.45.3, urllib3-2.2.3, websockets-13.1
[debug] Proxy map: {}
[debug] Extracting cookies from: "C:\Users\J\AppData\Roaming\Mozilla\Firefox\Profiles\c2ty66d6.default-release\cookies.sqlite"
[debug] Request Handlers: urllib, requests, websockets, curl_cffi
[debug] Loaded 1837 extractors
[debug] Loading archive file 'F:\\yt-dlp tiktok likes\\archive.txt'
[debug] Command-line config: ['https://www.tiktok.com/@smostervik/video/7434809831665503520', '--download-archive', 'F:\\yt-dlp tiktok likes\\archive.txt', '--write-thumbnail', '--embed-thumbnail', '--verbose', '--cookies-from-browser', 'firefox']
[debug] Encodings: locale cp1252, fs utf-8, pref cp1252, out utf-8, error cp1252 (No VT), screen utf-8
[debug] yt-dlp version stable@2024.11.18 from yt-dlp/yt-dlp [7ea278792] (pip)
[debug] Python 3.11.9 (CPython AMD64 64bit) - Windows-10-10.0.22631-SP0 (OpenSSL 3.3.2 3 Sep 2024)
[debug] exe versions: ffmpeg 7.0.2-full_build-www.gyan.dev (setts), ffprobe 7.0.2-full_build-www.gyan.dev
[debug] Optional libraries: Cryptodome-3.20.0, brotli-1.0.9, certifi-2024.08.30, curl_cffi-0.5.10, mutagen-1.47.0, requests-2.32.3, sqlite3-3.45.3, urllib3-2.2.3, websockets-13.1
[debug] Proxy map: {}
[debug] Extracting cookies from: "C:\Users\J\AppData\Roaming\Mozilla\Firefox\Profiles\c2ty66d6.default-release\cookies.sqlite"
[debug] Request Handlers: urllib, requests, websockets, curl_cffi
[debug] Loaded 1837 extractors
[debug] Loading archive file 'F:\\yt-dlp tiktok likes\\archive.txt'
[debug] Command-line config: ['https://www.tiktok.com/@bille.135/video/7439449253501603104', '--download-archive', 'F:\\yt-dlp tiktok likes\\archive.txt', '--write-thumbnail', '--embed-thumbnail', '--verbose', '--cookies-from-browser', 'firefox']
[debug] Encodings: locale cp1252, fs utf-8, pref cp1252, out utf-8, error cp1252 (No VT), screen utf-8
[debug] yt-dlp version stable@2024.11.18 from yt-dlp/yt-dlp [7ea278792] (pip)
[debug] Python 3.11.9 (CPython AMD64 64bit) - Windows-10-10.0.22631-SP0 (OpenSSL 3.3.2 3 Sep 2024)
[debug] exe versions: ffmpeg 7.0.2-full_build-www.gyan.dev (setts), ffprobe 7.0.2-full_build-www.gyan.dev
[debug] Optional libraries: Cryptodome-3.20.0, brotli-1.0.9, certifi-2024.08.30, curl_cffi-0.5.10, mutagen-1.47.0, requests-2.32.3, sqlite3-3.45.3, urllib3-2.2.3, websockets-13.1
[debug] Proxy map: {}
[debug] Extracting cookies from: "C:\Users\J\AppData\Roaming\Mozilla\Firefox\Profiles\c2ty66d6.default-release\cookies.sqlite"
[debug] Request Handlers: urllib, requests, websockets, curl_cffi
[debug] Loaded 1837 extractors
[debug] Loading archive file 'F:\\yt-dlp tiktok likes\\archive.txt'
[debug] Command-line config: ['https://www.tiktok.com/@kristal.329/video/7435311238092950815', '--download-archive', 'F:\\yt-dlp tiktok likes\\archive.txt', '--write-thumbnail', '--embed-thumbnail', '--verbose', '--cookies-from-browser', 'firefox']
[debug] Encodings: locale cp1252, fs utf-8, pref cp1252, out utf-8, error cp1252 (No VT), screen utf-8
[debug] yt-dlp version stable@2024.11.18 from yt-dlp/yt-dlp [7ea278792] (pip)
[debug] Python 3.11.9 (CPython AMD64 64bit) - Windows-10-10.0.22631-SP0 (OpenSSL 3.3.2 3 Sep 2024)
[debug] exe versions: ffmpeg 7.0.2-full_build-www.gyan.dev (setts), ffprobe 7.0.2-full_build-www.gyan.dev
[debug] Optional libraries: Cryptodome-3.20.0, brotli-1.0.9, certifi-2024.08.30, curl_cffi-0.5.10, mutagen-1.47.0, requests-2.32.3, sqlite3-3.45.3, urllib3-2.2.3, websockets-13.1
[debug] Proxy map: {}
[debug] Extracting cookies from: "C:\Users\J\AppData\Roaming\Mozilla\Firefox\Profiles\c2ty66d6.default-release\cookies.sqlite"
[debug] Request Handlers: urllib, requests, websockets, curl_cffi
[debug] Loaded 1837 extractors
[debug] Loading archive file 'F:\\yt-dlp tiktok likes\\archive.txt'
[debug] Command-line config: ['https://www.tiktok.com/@johanna_nordstrand/video/7440174704758983969', '--download-archive', 'F:\\yt-dlp tiktok likes\\archive.txt', '--write-thumbnail', '--embed-thumbnail', '--verbose', '--cookies-from-browser', 'firefox']
[debug] Encodings: locale cp1252, fs utf-8, pref cp1252, out utf-8, error cp1252 (No VT), screen utf-8
[debug] yt-dlp version stable@2024.11.18 from yt-dlp/yt-dlp [7ea278792] (pip)
[debug] Python 3.11.9 (CPython AMD64 64bit) - Windows-10-10.0.22631-SP0 (OpenSSL 3.3.2 3 Sep 2024)
[debug] exe versions: ffmpeg 7.0.2-full_build-www.gyan.dev (setts), ffprobe 7.0.2-full_build-www.gyan.dev
[debug] Optional libraries: Cryptodome-3.20.0, brotli-1.0.9, certifi-2024.08.30, curl_cffi-0.5.10, mutagen-1.47.0, requests-2.32.3, sqlite3-3.45.3, urllib3-2.2.3, websockets-13.1
[debug] Proxy map: {}
[debug] Extracting cookies from: "C:\Users\J\AppData\Roaming\Mozilla\Firefox\Profiles\c2ty66d6.default-release\cookies.sqlite"
[debug] Request Handlers: urllib, requests, websockets, curl_cffi
[debug] Loaded 1837 extractors
[debug] Loading archive file 'F:\\yt-dlp tiktok likes\\archive.txt'
[debug] Command-line config: ['https://www.tiktok.com/@cassidyannpayne/video/7440590041866456362', '--download-archive', 'F:\\yt-dlp tiktok likes\\archive.txt', '--write-thumbnail', '--embed-thumbnail', '--verbose', '--cookies-from-browser', 'firefox']
[debug] Encodings: locale cp1252, fs utf-8, pref cp1252, out utf-8, error cp1252 (No VT), screen utf-8
[debug] yt-dlp version stable@2024.11.18 from yt-dlp/yt-dlp [7ea278792] (pip)
[debug] Python 3.11.9 (CPython AMD64 64bit) - Windows-10-10.0.22631-SP0 (OpenSSL 3.3.2 3 Sep 2024)
[debug] exe versions: ffmpeg 7.0.2-full_build-www.gyan.dev (setts), ffprobe 7.0.2-full_build-www.gyan.dev
[debug] Optional libraries: Cryptodome-3.20.0, brotli-1.0.9, certifi-2024.08.30, curl_cffi-0.5.10, mutagen-1.47.0, requests-2.32.3, sqlite3-3.45.3, urllib3-2.2.3, websockets-13.1
[debug] Proxy map: {}
[debug] Extracting cookies from: "C:\Users\J\AppData\Roaming\Mozilla\Firefox\Profiles\c2ty66d6.default-release\cookies.sqlite"
[debug] Request Handlers: urllib, requests, websockets, curl_cffi
[debug] Loaded 1837 extractors
[debug] Loading archive file 'F:\\yt-dlp tiktok likes\\archive.txt'
[debug] Command-line config: ['https://www.tiktok.com/@backup_josefinelykk/video/7440092940057267488', '--download-archive', 'F:\\yt-dlp tiktok likes\\archive.txt', '--write-thumbnail', '--embed-thumbnail', '--verbose', '--cookies-from-browser', 'firefox']
[debug] Encodings: locale cp1252, fs utf-8, pref cp1252, out utf-8, error cp1252 (No VT), screen utf-8
[debug] yt-dlp version stable@2024.11.18 from yt-dlp/yt-dlp [7ea278792] (pip)
[debug] Python 3.11.9 (CPython AMD64 64bit) - Windows-10-10.0.22631-SP0 (OpenSSL 3.3.2 3 Sep 2024)
[debug] exe versions: ffmpeg 7.0.2-full_build-www.gyan.dev (setts), ffprobe 7.0.2-full_build-www.gyan.dev
[debug] Optional libraries: Cryptodome-3.20.0, brotli-1.0.9, certifi-2024.08.30, curl_cffi-0.5.10, mutagen-1.47.0, requests-2.32.3, sqlite3-3.45.3, urllib3-2.2.3, websockets-13.1
[debug] Proxy map: {}
[debug] Extracting cookies from: "C:\Users\J\AppData\Roaming\Mozilla\Firefox\Profiles\c2ty66d6.default-release\cookies.sqlite"
[debug] Request Handlers: urllib, requests, websockets, curl_cffi
[debug] Loaded 1837 extractors
[debug] Loading archive file 'F:\\yt-dlp tiktok likes\\archive.txt'
[debug] Command-line config: ['https://www.tiktok.com/@elina.pp3/video/7439466484176391456', '--download-archive', 'F:\\yt-dlp tiktok likes\\archive.txt', '--write-thumbnail', '--embed-thumbnail', '--verbose', '--cookies-from-browser', 'firefox']
```

--- a/yt_dlp/extractor/tiktok.py
+++ b/yt_dlp/extractor/tiktok.py
@@ -420,6 +420,7 @@ class TikTokBaseIE(InfoExtractor):
                 thumbnails.append({
                     'id': cover_id,
                     'url': cover_url,
+                    'preference': -1 if cover_id in ('cover', 'origin_cover') else -2,
                 })
 
         stats_info = aweme_detail.get('statistics') or {}
@@ -572,11 +573,21 @@ class TikTokBaseIE(InfoExtractor):
             'uploader_id': (('authorId', 'uid', 'id'), {str_or_none}),
         }), get_all=False)
 
+        thumbnails = []
+        for cover_id in ('thumbnail', 'cover', 'dynamicCover', 'originCover'):
+            for cover_url in traverse_obj(aweme_detail, ((None, 'video'), cover_id, {url_or_none})):
+                thumbnails.append({
+                    'id': cover_id,
+                    'url': self._proto_relative_url(cover_url),
+                    'preference': -2 if cover_id == 'dynamicCover' else -1,
+                })
+
         return {
             'id': video_id,
             'formats': None if extract_flat else self._extract_web_formats(aweme_detail),
             'subtitles': None if extract_flat else self.extract_subtitles(aweme_detail, video_id, None),
             'http_headers': {'Referer': webpage_url},
+            'thumbnails': thumbnails,
             **author_info,
             'channel_url': format_field(author_info, 'channel_id', self._UPLOADER_URL_FORMAT, default=None),
             'uploader_url': format_field(
@@ -600,11 +611,6 @@ class TikTokBaseIE(InfoExtractor):
                 'repost_count': 'shareCount',
                 'comment_count': 'commentCount',
             }), expected_type=int_or_none),
-            'thumbnails': traverse_obj(aweme_detail, (
-                (None, 'video'), ('thumbnail', 'cover', 'dynamicCover', 'originCover'), {
-                    'url': ({url_or_none}, {self._proto_relative_url}),
-                },
-            )),
         }
 
 

--- a/yt_dlp/extractor/tiktok.py
+++ b/yt_dlp/extractor/tiktok.py
@@ -413,16 +413,6 @@ class TikTokBaseIE(InfoExtractor):
             for f in formats:
                 self._set_cookie(urllib.parse.urlparse(f['url']).hostname, 'sid_tt', auth_cookie.value)
 
-        thumbnails = []
-        for cover_id in ('cover', 'ai_dynamic_cover', 'animated_cover', 'ai_dynamic_cover_bak',
-                         'origin_cover', 'dynamic_cover'):
-            for cover_url in traverse_obj(video_info, (cover_id, 'url_list', ...)):
-                thumbnails.append({
-                    'id': cover_id,
-                    'url': cover_url,
-                    'preference': -1 if cover_id in ('cover', 'origin_cover') else -2,
-                })
-
         stats_info = aweme_detail.get('statistics') or {}
         music_info = aweme_detail.get('music') or {}
         labels = traverse_obj(aweme_detail, ('hybrid_label', ..., 'text'), expected_type=str)
@@ -468,7 +458,17 @@ class TikTokBaseIE(InfoExtractor):
             'formats': formats,
             'subtitles': self.extract_subtitles(
                 aweme_detail, aweme_id, traverse_obj(author_info, 'uploader', 'uploader_id', 'channel_id')),
-            'thumbnails': thumbnails,
+            'thumbnails': [
+                {
+                    'id': cover_id,
+                    'url': cover_url,
+                    'preference': -1 if cover_id in ('cover', 'origin_cover') else -2,
+                }
+                for cover_id in (
+                    'cover', 'ai_dynamic_cover', 'animated_cover',
+                    'ai_dynamic_cover_bak', 'origin_cover', 'dynamic_cover')
+                for cover_url in traverse_obj(video_info, (cover_id, 'url_list', ...))
+            ],
             'duration': (traverse_obj(video_info, (
                 (None, 'download_addr'), 'duration', {int_or_none(scale=1000)}, any))
                 or traverse_obj(music_info, ('duration', {int_or_none}))),
@@ -573,21 +573,20 @@ class TikTokBaseIE(InfoExtractor):
             'uploader_id': (('authorId', 'uid', 'id'), {str_or_none}),
         }), get_all=False)
 
-        thumbnails = []
-        for cover_id in ('thumbnail', 'cover', 'dynamicCover', 'originCover'):
-            for cover_url in traverse_obj(aweme_detail, ((None, 'video'), cover_id, {url_or_none})):
-                thumbnails.append({
-                    'id': cover_id,
-                    'url': self._proto_relative_url(cover_url),
-                    'preference': -2 if cover_id == 'dynamicCover' else -1,
-                })
-
         return {
             'id': video_id,
             'formats': None if extract_flat else self._extract_web_formats(aweme_detail),
             'subtitles': None if extract_flat else self.extract_subtitles(aweme_detail, video_id, None),
             'http_headers': {'Referer': webpage_url},
-            'thumbnails': thumbnails,
+            'thumbnails': [
+                {
+                    'id': cover_id,
+                    'url': self._proto_relative_url(cover_url),
+                    'preference': -2 if cover_id == 'dynamicCover' else -1,
+                }
+                for cover_id in ('thumbnail', 'cover', 'dynamicCover', 'originCover')
+                for cover_url in traverse_obj(aweme_detail, ((None, 'video'), cover_id, {url_or_none}))
+            ],
             **author_info,
             'channel_url': format_field(author_info, 'channel_id', self._UPLOADER_URL_FORMAT, default=None),
             'uploader_url': format_field(

--- a/yt_dlp/extractor/tiktok.py
+++ b/yt_dlp/extractor/tiktok.py
@@ -578,15 +578,6 @@ class TikTokBaseIE(InfoExtractor):
             'formats': None if extract_flat else self._extract_web_formats(aweme_detail),
             'subtitles': None if extract_flat else self.extract_subtitles(aweme_detail, video_id, None),
             'http_headers': {'Referer': webpage_url},
-            'thumbnails': [
-                {
-                    'id': cover_id,
-                    'url': self._proto_relative_url(cover_url),
-                    'preference': -2 if cover_id == 'dynamicCover' else -1,
-                }
-                for cover_id in ('thumbnail', 'cover', 'dynamicCover', 'originCover')
-                for cover_url in traverse_obj(aweme_detail, ((None, 'video'), cover_id, {url_or_none}))
-            ],
             **author_info,
             'channel_url': format_field(author_info, 'channel_id', self._UPLOADER_URL_FORMAT, default=None),
             'uploader_url': format_field(
@@ -610,6 +601,15 @@ class TikTokBaseIE(InfoExtractor):
                 'repost_count': 'shareCount',
                 'comment_count': 'commentCount',
             }), expected_type=int_or_none),
+            'thumbnails': [
+                {
+                    'id': cover_id,
+                    'url': self._proto_relative_url(cover_url),
+                    'preference': -2 if cover_id == 'dynamicCover' else -1,
+                }
+                for cover_id in ('thumbnail', 'cover', 'dynamicCover', 'originCover')
+                for cover_url in traverse_obj(aweme_detail, ((None, 'video'), cover_id, {url_or_none}))
+            ],
         }
 
 

